esphome/components/tcs34725/tcs34725.cpp

```diff
@@ -254,7 +255,8 @@ void TCS34725Component::update() {
     // change integration time an gain to achieve maximum resolution an dynamic range
     // calculate optimal integration time to achieve 70% satuaration
     float integration_time_ideal;
-    integration_time_ideal = 60 / ((float) raw_c / 655.35) * this->integration_time_;
+
+    integration_time_ideal = 60 / ((float) std::max((uint16_t) 1, raw_c) / 655.35f) * this->integration_time_;
```

Comment:
Why not use a ternary operator for this simple use case, instead of having to add an additional include?
```suggestion
    integration_time_ideal = 60 / ((float) (raw_c ? raw_c : 1) / 655.35f) * this->integration_time_;
```

Reply:
Sure, if that's the preferred method?

Reply:
@guillempages Btw: Shouldn't be an import or a standard lib method be a noop anyway?